go.sum

```diff
@@ -962,8 +962,8 @@ google.golang.org/protobuf v1.24.0/go.mod h1:r/3tXBNzIEhYS9I1OUVjXDlt8tc493IdKGj
 google.golang.org/protobuf v1.25.0/go.mod h1:9JNX74DMeImyA3h4bdi1ymwjUzf21/xIlbajtzgsN7c=
 google.golang.org/protobuf v1.26.0-rc.1/go.mod h1:jlhhOSvTdKEhbULTjvd4ARK9grFBp09yW+WbY/TyQbw=
 google.golang.org/protobuf v1.26.0/go.mod h1:9q0QmTI4eRPtz6boOQmLYwt+qCgq0jsYwAQnmE0givc=
-google.golang.org/protobuf v1.32.0 h1:pPC6BG5ex8PDFnkbrGU3EixyhKcQ2aDuBS36lqK/C7I=
-google.golang.org/protobuf v1.32.0/go.mod h1:c6P6GXX6sHbq/GpV6MGZEdwhWPcYBgnhAHhKbcUYpos=
+google.golang.org/protobuf v1.33.0 h1:uNO2rsAINq/JlFpSdYEKIZ0uKD/R9cpdv0T+yoGwGmI=
+google.golang.org/protobuf v1.33.0/go.mod h1:c6P6GXX6sHbq/GpV6MGZEdwhWPcYBgnhAHhKbcUYpos=
```

Comment:
something is not right that it did not try to update `helm` :/

Reply:
@michaelbeaumont - any ideas why why helm is left at `helm.sh/helm/v3 v3.12.3` instead of bumping to `3.14.2` (which resolves this for example https://osv.dev/vulnerability/GHSA-r53h-jv2g-vpx6 )?

Reply:
FYI @lobkovilya https://github.com/kumahq/kuma/pull/9521

Reply:
I think the condition is wrong here https://github.com/kumahq/kuma/blob/a3495495ee13b87de468816d7517890e9f672135/tools/ci/update-vulnerable-dependencies.sh#L25

It will update only if each segment is greater or equal, but for 3.14.2 the patch version is less than 3.12.3